### PR TITLE
minicom: add dependency on lrzsz

### PR DESCRIPTION
**Pull request by tewarid**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

See https://github.com/orgs/Homebrew/discussions/3699.

### CODE REVIEW

File: Formula/lrzsz.rb [hunk 1]
```diff
@@ -62,6 +62,7 @@ def install
 
     system "make", "install"
     bin.install_symlink "lrz" => "rz", "lsz" => "sz"
+    bin.install_symlink "lsz" => "sb"
```
Comment (fxcoudert) [inline Formula/lrzsz.rb:65]: Why provide a symlink with a different name, if it is not provided by upstream?

Comment (tewarid) [reply Formula/lrzsz.rb:65]: Linux distros such as Debian provide these (sz, sb...) symlinks e.g. https://packages.debian.org/bullseye/amd64/lrzsz/filelist, so we'll need to check why they are not created by default by upstream.

Comment (tewarid) [reply Formula/lrzsz.rb:65]: Looking at `Specfile.in` upstream and it apparently does something like `strip $RPM_BUILD_ROOT/usr/bin/{rb,rx,rz,sb,sx,sz}` which means it does create these symlinks for RPM packaging.

Comment (tewarid) [reply Formula/lrzsz.rb:65]: Upstream build neither creates the sb symlink, nor the sz symlink created by this recipe already.

Comment (tewarid) [reply Formula/lrzsz.rb:65]: Nudge to keep PR open.

Comment (tewarid) [reply Formula/lrzsz.rb:65]: > Why provide a symlink with a different name, if it is not provided by upstream?

One symlink, sz, is created by this recipe already. sb program name makes lrzsz use the ymodem protocol. I don't know why upstream chose not to create these symlinks as part of its build.

**Comment by tewarid** [comment]
Still need this for minicom to use ymodem.

**Pull request closed without merging**